src/auto-evo/RunResults.cs

```diff
@@ -205,35 +211,31 @@ void OutputPopulationForPatch(Species species, Patch patch, int population)
 
                 if (entry.SpreadToPatches.Count > 0)
                 {
-                    builder.Append(" spread to patches:\n");
+                    builder.Append(" ");
+                    builder.Append(TranslationServer.Translate("RUNRESULT_SPREAD_PATCHES"));
 
                     foreach (var spreadEntry in entry.SpreadToPatches)
                     {
-                        if (playerReadable)
-                        {
-                            builder.Append("  ");
-                            builder.Append(spreadEntry.To.Name);
-                            builder.Append(" by sending: ");
-                            builder.Append(spreadEntry.Population);
-                            builder.Append(" population");
-                            builder.Append(" from patch: ");
-                            builder.Append(spreadEntry.From.Name);
-                        }
-                        else
-                        {
-                            builder.Append("  ");
-                            builder.Append(spreadEntry.To.Name);
-                            builder.Append(" pop: ");
-                            builder.Append(spreadEntry.Population);
-                            builder.Append(" from: ");
-                            builder.Append(spreadEntry.From.Name);
-                        }
-
+                        builder.Append("  ");
+                        builder.Append(spreadEntry.To.Name);
+                        builder.Append(" ");
+                        builder.Append(playerReadable ?
+                            TranslationServer.Translate("RUNRESULT_BY_SENDING") :
+                            TranslationServer.Translate("RUNRESULT_POP_SHORT"));
+                        builder.Append(" ");
+                        builder.Append(spreadEntry.Population);
+                        builder.Append(" ");
+                        builder.Append(playerReadable ?
+                            TranslationServer.Translate("RUNRESULT_POP_FROM_PATCH") :
+                            TranslationServer.Translate("RUNRESULT_FROM"));
+                        builder.Append(" ");
+                        builder.Append(spreadEntry.From.Name);
```

Comment:
This should probably also be a single template string in the translations.

Reply:
If playerReadable flag is false, that doesn't need to be translatable, it would even be preferrable if that wasn't as that is for debugging and log printing purposes only.